app/serializers/job_for_company_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class JobForCompanySerializer < ActiveModel::Serializer
+  attributes :id, :title, :description, :url, :location, :posted_date, :remote
+  has_many :technologies
```

Comment:
Just a style comment. It can be a bit easier to read if the attributes are listed on separate lines. Not a huge deal with a short list, but can get difficult to read as the attributes list increases:
```
attributes :id,
                :title,
                :description,
                :url,
                ....
```